_projects/nested-languages.md

```diff
@@ -24,16 +24,101 @@ tags:
   - Core
   - Bears
 ---
-Multiple programming languages can coexist in a single source file. coala would
+Generally, a single source file contains code for exactly one programming language.
+But multiple programming languages can also coexist in a single source file. coala would
 benefit from the ability to determine the programming language of each part of
 a file. If we have e.g. a PHP file, it is likely that we'd have chunks of html
 inside a few of them. We would like coala to be able to still run meaningful
 analysis on those files.
 Other examples include:
 
-* awk inside bash
-* JSON inside Javascript
-* HTML, LaTeX or others inside Jinja2
+* Python inside PHP
```

Comment:
please show me a real example of Python inside PHP?

nested language support is about real world nested languages.

This project is not about creating new language-inside-language runtime engines.

Reply:
@jayvdb  that means this example is not suitable here?

Reply:
fake examples are not helpful to let me understand what this is.

there are so many real examples, fake ones are unnecessary.